packages/ui-extensions/package.json

```diff
@@ -67,7 +67,7 @@
     "@remote-ui/core": "^2.2.4"
   },
   "devDependencies": {
-    "@shopify/generate-docs": "0.16.0",
```

Comment:
The [Prepare UI extension release](https://github.com/Shopify/checkout-web/actions/workflows/prepare-ui-extension-package-release.yml) Github Action didn't automatically update this change, even though it was [updated in `checkout-web` ](https://github.com/Shopify/checkout-web/blob/d6b3be9d3a9b176e408ce0fac8566d10b358b808/packages/ui-extensions/package.json#L35-L38)

Reply:
I believe it's ok, since this is an exact copy of the package, and will be used to generate the docs correctly from ui-extensions, it should be ok.

unless you have any concerns here @jamesvidler ?

Reply:
What you did here looks right, Hayley 👍 We needed to update `generate-docs` to get the [type fix](https://github.com/Shopify/checkout-web/issues/37106).